src/processor.ts

```diff
@@ -145,6 +145,11 @@ processor.run(new TypeormDatabase(), async (ctx) => {
     // Fetch max block height from the archive
     let maxHeight = await maxHeightPromise
 
+    // Determine the tallest block in the batch
+    let tallestBlockHeight = Math.max(
```

Comment:
'tallest' sounds weird haha. Can we rename it to `highestBlockHeight` or similar? Tall sounds so human.
Maybe it's actually related to check if `ctx.isHead` in the condition below and not checking the block height. Should already improve it a lot, WDYT @bogdanS98?

Reply:
Indeed it would improve by a lot, but `ctx.isHead` will always be true after the endpoint switches from archive to RPC (real-time) because every new batch that comes through will contain the head. So if we don't check that the block we're currently processing is indeed the head (has the biggest height) then we would always save the prices for every block in the batch.  We could run it with and without the second condition to see if the difference is significant, how does that sound to you @ebma?

Reply:
But I assume that once we are connected to the RPC (ie are getting blocks in real-time) then the batch would always just contain 1 block (?) And if that's the case, we don't need to check after all. 

Can you please check the batch sizes that we get once connected to the RPC? Then we base our decision on that info.

Reply:
Logged the batch size and you're right, after switching to RPC the batch always contains only the latest block so we can remove the additional condition
![image](https://github.com/pendulum-chain/pendulum-squids/assets/10613980/90613115-df11-4f42-a2ae-2bec5f501487)

Reply:
Updated the code 👍🏼